src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -16,13 +16,23 @@ public partial class InvokeRestMethodCommand
         /// <summary>
         /// gets or sets the parameter Method
         /// </summary>
-        [Parameter]
+        [Parameter(ParameterSetName = "StandardMethod")]
         public override WebRequestMethod Method
         {
             get { return base.Method; }
             set { base.Method = value; }
         }
 
+        /// <summary>
+        /// gets or sets the parameter CustomMethod
+        /// </summary>
+        [Parameter(ParameterSetName = "CustomMethod")]
```

Comment:
You can make this validate on input and removed the checks for ```IsNullOrEmpty```. Also adding and alias now would be nice.

[ValidateNotNullOrEmpty]
[Alias("CM")]